src/main/java/com/arpnetworking/metrics/common/sources/FileSource.java

```diff
@@ -301,6 +395,31 @@ public Builder() {
         }
 
         /**
+         * Sets the number of threads to be used to process lines read from the file. Optional.
+         * Default is Runtime.getRuntime().availableProcessors().
+         *
+         * @param value The number of threads to be used.
+         * @return This instance of <code>Builder</code>.
+         */
+        public final Builder<T> setLineProcessorThreads(final int value) {
```

Comment:
One of the requirements we discussed was supporting "2C" which is two times cores, or more generally "{N}C". Please revise.